### PR TITLE
adding new author ychen

### PR DESCRIPTION
**Pull request by chen2055**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/ychen.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Yan Chen
+short_name: Yan
+twitter:
```
Comment (joshuarrrr) [inline _authors/ychen.markdown:4]: Remove this line if not needed
```suggestion
```

**Review by joshuarrrr — CHANGES_REQUESTED** [review]
@chen2055 Please move the new file to the `_authors/osci_2023` directory.

File: _authors/ychen.markdown (hunk 1 above)
Comment (chen2055) [reply _authors/ychen.markdown]: Thanks for the feedback. I committed the changes accordingly. Do I need to open a new pull request?

**Review by dtaivpp — APPROVED** [review]
LGTM!

**Pull request merged**